### ISSUE TITLE
apperance of the pretrained model

### ISSUE THREAD

**Issue by lyupei** [preritj/pose_estimation#1]
I tested the pretrained model using the inference.py(added padding to avoid the distortion of image) with coco val images and ai_challenge images, the result seems abnormal even in most easy cases. Only very few images works well.

**Comment by preritj** [comment]
The pre-trained model is fine-tuned for cameras mounted on top while most coco images are front-view. Unfortunately, I do not have model trained on coco dataset, so closing this issue. Will update ReadMe to reflect this. Contributions are welcome.

**Comment by SmallLogicer** [comment]
Thanks for your work @preritj, want to know is this dataset public available or a private dataset, if it is open sourced dataset, could you please give me the link of it?

**Comment by preritj** [comment]
Unfortunately no, it is private dataset

**Comment by SmallLogicer** [comment]
Cool, I got it, thank you!
